### DISCUSSION TITLE
How to integrate RMT with Conformer

### DISCUSSION THREAD

**Discussion by pfeatherstone** [lucidrains/recurrent-memory-transformer-pytorch#21] [General]
If i were to integrate RMT inside Conformer, would there be any extra states, e.g. from the convolutional layers, that would have to be cached for the next time steps. I can't see how one would do that. Or would caching the memories from RMT be sufficient?

**Comment by lucidrains** [comment]
@pfeatherstone the memories need to be excised before the convolutions. the convolutions will also need to be causal

it is all around tricky and requires non trivial engineering work
